README.md

```diff
@@ -117,6 +117,19 @@ In order to access the admin features of the TestTrack server, you must create a
 > Admin.create!(email: "myemail@example.org", password: "[something secret]")
 ```
 
+### Enabling attachments
+Variants can be associated with metadata to describe their effects in human-readable terms. To enable variant screenshots, you'll need some additional configuration in environment variables.
+
+- To store uploads on the local file system, set `ATTACHMENT_STORAGE` to `local`, and:
+  - `LOCAL_UPLOAD_PATH` (optional, defaults to `:rails_root/public/system/:class/:attachment/:id_partition/:style/:filename`)
+- To store uploads in S3, set `ATTACHMENT_STORAGE` to `s3`, and:
+  - `S3_ACCESS_KEY_ID` (required)
+  - `S3_SECRET_ACCESS_KEY` (required)
```

Comment:
Hmmm, I'm just realizing that we should maybe be using the AWS default Env var names for these? If you use their defaults, the lib doesn't need to be passed them at all. Especially in a single-purpose app like TestTrack it doesn't make much sense to have multiple different IAM identities associated with the app, but I believe in general there's no need for an app to have more than one identity either. Can we make these the expected AWS client default var names?